testnets/osmosistestnet/assetlist.json

```diff
@@ -94,6 +94,115 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/osmosistestnet/images/willyz.svg"
         }
       ]
+    },
+    {
+      "description": "The native EVM, governance and staking token of the Titan Chain",
+      "denom_units": [
+        {
+          "denom": "ibc/28EEF762052DB8C3D27A7BF625E9F86A1B3B689CC8D80C818CEDF2EB6CBA02A6",
+          "exponent": 0,
+          "aliases": [
+            "atkx"
+          ]
+        },
+        {
+          "denom": "utkx",
+          "exponent": 12,
+          "aliases": [
+            "microtkx"
+          ]
+        },
+        {
+          "denom": "mtkx",
+          "exponent": 15,
+          "aliases": [
+            "millitkx"
+          ]
+        },
+        {
+          "denom": "tkx",
+          "exponent": 18
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/28EEF762052DB8C3D27A7BF625E9F86A1B3B689CC8D80C818CEDF2EB6CBA02A6",
+      "name": "tkx",
+      "display": "tkx",
+      "symbol": "TKX",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "titan",
+            "base_denom": "atkx",
+            "channel_id": "channel-2"
+          },
+          "chain": {
+            "channel_id": "channel-5969",
+            "path": "transfer/channel-5969/atkx"
+          }
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/titantestnet/images/tkx.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/titantestnet/images/tkx.png"
+        }
+      ]
+    },
+    {
+      "description": "Circle's stablecoin on Axelar",
+      "denom_units": [
+        {
+          "denom": "ibc/1587E7B54FC9EFDA2350DC690EC2F9B9ECEB6FC31CF11884F9C0C5207ABE3921",
+          "exponent": 0
+        },
+        {
+          "denom": "ausdc",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/1587E7B54FC9EFDA2350DC690EC2F9B9ECEB6FC31CF11884F9C0C5207ABE3921",
+      "name": "USD Coin (Axelar)",
+      "display": "ausdc",
+      "symbol": "aUSDC.axl",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "ethereumtestnet",
+            "base_denom": "0x254d06f33bDc5b8ee05b2ea472107E300226659A"
+          },
+          "provider": "Axelar"
+        },
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "axelartestnet",
+            "base_denom": "uausdc",
+            "channel_id": "channel-521"
+          },
+          "chain": {
+            "channel_id": "channel-4118",
```

Comment:
This connection between axelartestnet and osmosistestnet is not registered. Would need to register this to replace the old (currently registered) connection

Reply:
Thank u. I have changed back to use old connection.